core/src/pages/governance.astro

```diff
@@ -0,0 +1,315 @@
+    >
+      Nix Leadership Bodies
+    </h2>
+    <p class="mb-6">The Nix community is guided by two leadership bodies:</p>
+    <div class="mt-2 mb-3 grid gap-4 md:grid-cols-2">
+      <div
+        class="bg-secondary-afghani-blue-95 flex w-full flex-col gap-2 rounded-xl p-4"
+      >
+        <h3
+          class="font-heading text-secondary-afghani-blue text-3xl leading-none font-bold"
+        >
+          The Steering Committee
+        </h3>
+        <span class="text-secondary-afghani-blue-35 text-lg">
+          7 elected individuals to represent the community
+        </span>
+        <span>
+          <b>Responsibilities:</b> Technical and community decisions and delegation
+        </span>
+        <span>
+          <b>Members:</b>
+          {
+            steeringCommittee.data.members
+              .map((member) => member.name)
+              .join(', ')
+          }
+        </span>
+        <a href="/community/teams/steering-committee">More Info</a>
+      </div>
+      <div
+        class="bg-secondary-afghani-blue-95 flex w-full flex-col gap-2 rounded-xl p-4"
+      >
+        <h3
+          class="font-heading text-secondary-afghani-blue text-3xl leading-none font-bold"
+        >
+          The Foundation Board
+        </h3>
+        <span class="text-secondary-afghani-blue-35 text-lg">
+          Board members of the Stichting NixOS Foundation, a non-profit
+          organisation in Utrecht, the Netherlands
+        </span>
+        <span>
+          <b>Responsibilities:</b> Legal, financial and partnership decisions and
+          delegation
+        </span>
+        <span>
+          <b>Members:</b>
+          {foundationBoard.data.members.map((member) => member.name).join(', ')}
+        </span>
+        <a href="/community/teams/foundation-board">More Info</a>
+      </div>
+    </div>
+    <p>
+      Behind these bodies are many volunteers who contribute to the project in
+      various ways. Read more about our {teams.length} teams on the <a
+        href="/community">community page</a
+      >.
+    </p>
+    <h3
+      class="font-heading text-secondary-afghani-blue mt-4 text-3xl font-bold"
+    >
+      Governance Constitution
+    </h3>
+    <p class="mb-2">
+      The <a href="https://github.com/NixOS/org/blob/main/doc/constitution.md"
+        >Nix Governance Constitution</a
+      > describes each bodies responsibilities, how the steering committee is elected,
+      and how decisions are made inside the steering committee.
+    </p>
+    <h3
+      class="font-heading text-secondary-afghani-blue mt-4 text-3xl font-bold"
+    >
+      Steering Committee Elections
+    </h3>
+    <p class="mb-2">
+      Every year, the Nix community elects members to the steering committee.
+      Each member holds a two-year term, with elections held annually to fill
+      half of the seats. You can read more about the election process in the <a
```

Comment:
@infinisil I think you meant like this?
```suggestion
      at least half of the seats. You can read more about the election process in the <a
```

Reply:
Yes thanks! I think I tested some new review UI that was kind of broken, because I'm pretty sure I selected both lines for the suggestion 😆